server/@types/activeAgencies.ts

```diff
@@ -4,6 +4,7 @@ export enum ServiceName {
   ACTIVITIES = 'activities',
   CAS2 = 'cas2',
   ALERTS = 'alerts',
+  CSIP = 'csipUI',
```

Comment:
This value should match the `application` value in `getReleaseStatus`, i.e `csipApi`